README.md

```diff
@@ -0,0 +1,49 @@
+# Leantime OmniSearch
+
+A plugin for Leantime that gives access to a plethora of shortcuts, to speed up 
+navigating the system.
+
+## Development
+
+This plugin requires an instance of Leantime running locally.
+
+Git clone this repo into app/Plugins as follows:
+
+```shell
+    cd app/Plugins
+    git clone https://github.com/ITK-Leantime/leantime-omnisearch /OmniSearch
```

Comment:
You should not indent code inside fences code blocks (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks).
```suggestion
cd app/Plugins
git clone https://github.com/ITK-Leantime/leantime-omnisearch /OmniSearch
```